package/index.ts

```diff
@@ -421,7 +421,7 @@ export const isArray = Array.isArray;
 
 var isArrayVdom = isArray;
 
-export function setIsArrayVdom(isArrayFnc: (arg: any) => arg is any[]) {
+export function setIsArrayVdom(isArrayFnc: <T>(arg: T | {}) => arg is T extends readonly any[] ? (unknown extends T ? never : readonly any[]) : any[]) {
```

Comment:
typeof isArray by byl mozna lepsi i pro pripadne budouci zmeny?

Reply:
taky me to napadlo, tady by se pak nemuselo asi nic resit... ale udelal jsem to po vzoru puvodniho, ze jsem tam zadal type z d.ts ... otazka je, jaka byla motivace @Bobris tam vypsat cely typ

Reply:
No necekal jsem ze to zmeni a ten ten puvodni typ byl dostatecne jednoduchy, tohle uz je trochu nightmare.